kong.conf.default

```diff
@@ -521,6 +532,17 @@
                                              # This value is only used during
                                              # migrations.
 
+#declarative_config =           # The path to the declarative configuration
+                                # file which holds the specification of all
+                                # entities (Routes, Services, Consumers, etc.)
+                                # to be used when Kong runs without a database.
```

Comment:
Should we specify that `database=off` should be in effect for this property to have a meaningful impact?

Reply:
I thought about it, but we don't do it for the Postgres or Cassandra options (though those are more obviously prefixed, granted). It does spell out "when Kong runs without a database", though, so my take was that it would feel a bit redundant.

Reply:
The feature is relatively new, and users may not even be aware of how Kong needs to be configured to run without a database yet. Or rather, any new user even in the future may not be immediately aware that Kong can run without a database, and how. I wouldn't call it redundant if we reminded users how to do so here, especially since it is a requirement for this property to take effect. In other properties (including the other two edited in this patch) we aren't shying away from mentioning sibling properties (e.g. "ensure the in-memory cache is large enough via the `mem_cache_size` property).

Reply:
:+1: